langchain4j-azure-open-ai/src/main/java/dev/langchain4j/model/azure/AzureOpenAiStreamingChatModel.java

```diff
@@ -501,6 +558,11 @@ public Builder logRequestsAndResponses(boolean logRequestsAndResponses) {
             return this;
         }
 
+        public Builder useAsyncClient(boolean useAsyncClient) {
```

Comment:
I am wondering, is there a reason why would users need to use sync client in `AzureOpenAiStreamingChatModel`? I can't find one...

I would make `useAsyncClient(boolean)` and  `openAIClient(OpenAIClient)` `@Deprecated` and remove them and all sync client logic after a few versions

Reply:
done, do not see any valid reason as well :) AzureOpenAiChatModel should be used instead, if syncClient is preferred.